shrubbery/private/simple-pretty.rkt

```diff
@@ -70,6 +71,9 @@
              (loop (cadr doc) col indent measure-mode k)]
             [else
              (loop (caddr doc) col indent measure-mode k)])])]
+      [(eq? (car doc) 'special)
```

Comment:
This should have `(assume (= (length doc) 3))`?

Reply:
Granting that using a list for this internal representation is a little hacky, left as-is.